optuna/multi_objective/study.py

```diff
@@ -118,6 +118,45 @@ def load_study(
 ) -> "multi_objective.study.MultiObjectiveStudy":
     """Load the existing :class:`MultiObjectiveStudy` that has the specified name.
 
+    Example:
+
+        .. testsetup::
+
+                import os
+
+                if os.path.exists("example.db"):
+                    raise RuntimeError("'example.db' already exists. Please remove it.")
```

Comment:
The indent seems too deep. Could you reduce and align them?
```suggestion
            import os

            if os.path.exists("example.db"):
                raise RuntimeError("'example.db' already exists. Please remove it.")
```

Reply:
Thank you as always!